### ISSUE TITLE
Package: Attention

### ISSUE THREAD

**Issue by louisfd** [tracel-ai/cubecl#21]
Implement attention algorithms:
- [ ] Flash attention 2
- [ ] Flash attention 3
- [ ] Other attention algorithms

**Comment by cBournhonesque** [comment]
Those would be added as kernels in burn-jit, but using the cube macro?

**Comment by nathanielsimard** [comment]
@cBournhonesque yes, we are not sure where we will put the kernel implementations though. For now matmul is in `cubecl-linalg`, but we might create other themed crates.
